app/controllers/meetings_controller.rb

```diff
@@ -0,0 +1,73 @@
+class MeetingsController < ApplicationController
+  before_action :set_meeting, only: [:show, :edit, :update, :destroy]
+  before_action :authenticate_user!
+
+  def index
+    @meetings = Meeting.all
+  end
+
+  def show
+  end
+
+  def new
+    @meeting = Meeting.new
+  end
+
+  def edit
+  end
+
+  def create
+    # u = User.find_by(:id => params[:user_id])
+    # @meeting = Meeting.new(meeting_params)
+    @meeting = current_user.meetings.build(meeting_params)
```

Comment:
We need to add validations before creations to prevent any class between two interviews right?